src/main/kotlin/uk/gov/justice/digital/hmpps/prisonestate/jpa/PrisonRepository.kt

```diff
@@ -21,7 +21,7 @@ data class Prison(
 
   @OneToOne
   @JoinColumn("prison_id")
-  val gpPractice: PrisonGpPractice? = null
+  var gpPractice: PrisonGpPractice? = null
```

Comment:
needed since the practice isn't in the constructor, maybe I should add another one with it in...